src/inverseRobot.js

```diff
@@ -24,7 +24,28 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const arr1 = Object.keys(robot);
+  const arr2 = Object.values(robot);
+
+  for (let x = 0; x < arr1.length; x++) {
+    if (arr1.indexOf(arr1[x]) !== arr1.lastIndexOf(arr1[x])) {
+      return null;
+    }
+  }
+
+  for (let x = 0; x < arr2.length; x++) {
+    if (arr2.indexOf(arr2[x]) !== arr2.lastIndexOf(arr2[x])) {
+      return null;
+    }
+  }
+
```

Comment:
[CODE STYLE] - use correct check for property presence in object. Some properties can be present, but still contain falsy value.
GOOD EXAMPLE:

if (key in robot) {
EVEN BETTER EXAMPLE:

if (robot.hasOwnProperty(key)) {
BAD EXAMPLE:

if (robot[key] === undefined) {

if (robot[key]) {